Tasks/kudaka/Task2/task2.py

```diff
@@ -0,0 +1,181 @@
+errors = ["!", "'", '"', "£", "$", "%", "&", " ", "?", "^", "°", "_", ",", ";", ":"]
+while True:
+    try:
+        user_equation = input("PLEASE ENTER EQUATION IN FORMAT 'Y = KX + B' OR Y = KX:\n")
+        for error in errors:
+            if error in user_equation:
+                user_equation = user_equation.replace(error, "")
+        x_value = input("Please enter value for X ")
+        if "*" not in user_equation:
+            user_equation = user_equation.replace("x","*"+x_value)
+            replace_1 = user_equation[2::]
+            print(f"Your equation is {user_equation}\nx is {x_value}\nYour result of equation is:{eval(replace_1)}")
+        else:
+            user_equation = user_equation.replace("x",x_value)
+            replace_1 = user_equation[2::]
+            print(f"Your equation is: {user_equation}\nx is: {x_value}\nYour result of equation is:{eval(replace_1)}")
+    except Exception:
+        print("SOMETHING WENT WRONG PLEASE TRY AGAIN")
+    else:
+        break
+
+
+
+
+
+
+# "TRY THIS ONE:" 
+#      "IT'S HAS 3 VALUES TO INPUT AND MORE PRECISE FOR ERRORS"
```

Comment:
of course the more separate values we ask from user, the more control we have over validation, but it's better for user experience to input all the data in one batch